### PR TITLE
Pull Request for DPEL Website Files

### PR DESCRIPTION
**Pull request by cecilia-dwa**
Signed-off-by: Cecilia Tran <cecilia.tran@dreamworks.com>

Requesting to merge from DW's forked copy of the repo to the main ASWF repo for the DPEL website.

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'astro/config';
+
+import tailwind from "@astrojs/tailwind";
+
+// https://astro.build/config
+export default defineConfig({
+  site: 'https://dreamworksanimation.github.io',
```
Comment (ee33) [inline astro.config.mjs:7]: Should this be updated to aswf?

**Review by ee33 — COMMENTED** [review]
This is excellent. Love the how-to instructions, love the directory structure. I haven't read the whole thing yet. One question below.

**Comment by ee33** [comment]
Do we need both /images and /public/images?

File: astro.config.mjs (hunk 1 above)
Comment (ee33) [reply astro.config.mjs:7]: Cool

Comment (cecilia-dwa) [reply astro.config.mjs:7]: After talking with Matthew I will be updating this to the new site under aswf

**Comment by cecilia-dwa** [comment]
> Do we need both /images and /public/images?

I have removed the extra images folder not needed.

File: src/pages/asc-stem2/asc-stem2-license/index.astro
Comment (ee33) [inline src/pages/asc-stem2/asc-stem2-license/index.astro:1]: Empty file, is that ok?

**Review by ee33 — APPROVED** [review]
Ready to commit, and enable Github Pages.
(One question below, that can be fixed now or later.)

File:  [hunk 2]
```diff
@@ -0,0 +1,19 @@
+update to home page image paths
+
+Signed-off-by: Cecilia Tran <cecilia.tran@dreamworks.com>
+
+# Please enter the commit message for your changes. Lines starting
```
Comment (jminor) [inline :5]: This file looks like it was committed accidentally. It's filename is all whitespace, and seems to be a commit message?

Comment (jminor) [reply :5]: To be clear, this is harmless and we could remove it later. No need to block landing this PR.

**Review by jminor — APPROVED** [review]

File: src/pages/asc-stem2/asc-stem2-license/index.astro
Comment (jminor) [reply src/pages/asc-stem2/asc-stem2-license/index.astro:1]: Marking this as resolved for @ee33 (he doesn't have that button)

**Review by scmcduffee — APPROVED** [review]

**Pull request merged**